src/OSPSuite.Core/Domain/Services/IPKValuesCalculator.cs

```diff
@@ -12,16 +13,15 @@ public interface IPKValuesCalculator
       /// </summary>
       /// <param name="time">Time array</param>
       /// <param name="concentration">Concentration array (should have same length as time</param>
-      /// <param name="options"></param>
-      PKValues CalculatePK(IReadOnlyList<float> time, IReadOnlyList<float> concentration, PKCalculationOptions options = null);
+      /// <param name="options">Optional pk calculation options</param>
+      /// <param name="dynamicPKParameters">Optional list of dynamic parameters that will be calculated for the provided values</param>
+      PKValues CalculatePK(IReadOnlyList<float> time, IReadOnlyList<float> concentration, PKCalculationOptions options = null, IReadOnlyList<DynamicPKParameter> dynamicPKParameters = null);
 
       /// <summary>
       ///    Calculates and returns the pk values for the given [time, concentration] profile. If the dose is defined
       ///    global pk values such as Vss, CL, VD will be available
       ///    InfusionTime should only be set for a single dosing iv application
       /// </summary>
-      PKValues CalculatePK(DataColumn dataColumn, PKCalculationOptions options = null);
+      PKValues CalculatePK(DataColumn dataColumn, PKCalculationOptions options = null, IReadOnlyList<DynamicPKParameter> dynamicPKParameters = null);
```

Comment:
Passing a list of dynamic parameters that are calculated on top of the standard parameters